src/Pages/geonwoojeon/Login/Login.js

```diff
@@ -44,7 +94,10 @@ class LoginWoo extends React.Component {
               id="facebooke-login"
             >
               <span>
-                <img src="images/icon/facebook.png" alt="facebook" />
+                <img
+                  src="/images/geonwoojeon/images/icon/facebook.png"
+                  alt="facebook"
```

Comment:
alt속성이 src보다 좀 더 중요하다고 판단해서 alt를 먼저 기입해주는 컨벤션이 있습니다!
적용해주세요!
```suggestion
                <img
                  alt="facebook"
                  src="/images/geonwoojeon/images/icon/facebook.png"
```

Reply:
감사합니다 적용완료했습니다!!